<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgBuscarCliente</class>
 <widget class="QDialog" name="dlgBuscarCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Buscar Cliente</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>492</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cliente:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtCliente">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>8</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGuardarCliente">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>496</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBuscarClientes">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>5</y>
     <width>84</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Buscar Clientes</string>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/buscar.png</normaloff>:/buscar.png</iconset>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableClienteWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>96</y>
     <width>391</width>
     <height>139</height>
    </rect>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nombre</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Dirección</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Calle</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Altura</string>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>246</y>
     <width>295</width>
     <height>235</height>
    </rect>
   </property>
   <property name="title">
    <string>Cliente Seleccionado</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>75</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Docenas:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSelClienteId">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>22</y>
      <width>43</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>24</y>
      <width>43</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Id:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSelClienteNombre">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>46</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>48</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nombre:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>104</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nro Pao:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNroPao">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>102</y>
      <width>37</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkEsRegalo">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>150</y>
      <width>85</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Es Regalo:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sboxDocenas">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>72</y>
      <width>43</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.500000000000000</double>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkTienePedido">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>128</y>
      <width>103</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Tiene Pedido:</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>176</y>
      <width>79</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Observación:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSelClienteObservacion">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>174</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>204</y>
      <width>79</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Horario:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtHorario">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>203</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnBuscarClientesPorDireccion">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>35</y>
     <width>84</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Buscar Clientes</string>
   </property>
   <property name="text">
    <string>Por Dirección</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/buscar.png</normaloff>:/buscar.png</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDireccion">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>37</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtCalle">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>67</y>
     <width>127</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtAltura">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>67</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBuscarClientesPorCalleAltura">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>65</y>
     <width>84</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Buscar Clientes</string>
   </property>
   <property name="text">
    <string>Por Calle Altura</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/buscar.png</normaloff>:/buscar.png</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>38</y>
     <width>61</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dirección:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>66</y>
     <width>85</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Calle y Altura:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtCliente</tabstop>
  <tabstop>txtDireccion</tabstop>
  <tabstop>btnBuscarClientesPorDireccion</tabstop>
  <tabstop>txtCalle</tabstop>
  <tabstop>txtAltura</tabstop>
  <tabstop>btnBuscarClientesPorCalleAltura</tabstop>
  <tabstop>txtNroPao</tabstop>
  <tabstop>txtSelClienteId</tabstop>
  <tabstop>tableClienteWidget</tabstop>
  <tabstop>txtSelClienteNombre</tabstop>
  <tabstop>btnGuardarCliente</tabstop>
  <tabstop>btnBuscarClientes</tabstop>
  <tabstop>chkEsRegalo</tabstop>
  <tabstop>sboxDocenas</tabstop>
  <tabstop>chkTienePedido</tabstop>
  <tabstop>txtSelClienteObservacion</tabstop>
  <tabstop>txtHorario</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dlgBuscarCliente</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dlgBuscarCliente</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
